openmetadata-ui/src/main/resources/ui/src/components/DashboardDetails/DashboardDetails.component.tsx

```diff
@@ -849,19 +840,6 @@ const DashboardDetails = ({
           onCancel={onThreadPanelClose}
         />
       ) : null}
-      {selectedField ? (
-        <RequestDescriptionModal
-          createThread={createThread}
-          defaultValue={getDefaultValue(owner as EntityReference)}
-          header="Request description"
-          threadLink={getEntityFeedLink(
-            EntityType.DASHBOARD,
-            dashboardFQN,
-            selectedField
-          )}
```

Comment:
Why we just removed it? It wasn't needed anymore?

Reply:
It shouldn't have been removed. I'll revert this.